tide_publication.install

```diff
@@ -664,3 +664,23 @@ function tide_publication_update_8011() {
   }
   $config->set('settings', $settings)->save();
 }
+
+/**
+ * Removed featured_news option from the landing_page widget.
+ */
+function tide_publication_update_8012() {
+  $field = FieldConfig::loadByName('node', 'publication_page', 'field_landing_page_component');
+  if ($field) {
+    $handler_settings = $field->getSetting('handler_settings');
+    if (isset($handler_settings['target_bundles'])) {
+      if (isset($handler_settings['target_bundles']['featured_news'])) {
+        unset($handler_settings['target_bundles']['featured_news']);
+      }
+      if (isset($handler_settings['target_bundles']['featured_news'])) {
```

Comment:
This condition looks the same like you have in #L676.

Reply:
oh~ Thanks, changed.